src/PhpSpec/Formatter/JsonFormatter.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Chris Kruining <chrise@keruining.eu>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace PhpSpec\Formatter;
+
+use PhpSpec\Event\ExampleEvent;
+use PhpSpec\Event\SpecificationEvent;
+use PhpSpec\Event\SuiteEvent;
+
+final class JsonFormatter extends ConsoleFormatter
```

Comment:
this does not seem to use any of the helper of ConsoleFormatter (except the fact that it has a reference to the IO). So should it actually extend it ?